code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -48,6 +48,7 @@
 	var/armour_penetration = 0 //How much armour they ignore, as a flat reduction from the targets armour value
 	var/melee_damage_type = BRUTE //Damage type of a simple mob's melee attack, should it do damage.
 	var/list/damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 1, CLONE = 1, STAMINA = 0, OXY = 1) // 1 for full damage , 0 for none , -1 for 1:1 heal from that source
+	var/list/armor = list(BRUTE = 0, BURN = 0, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
```

Comment:
Doesn't this make them invulnerable?

Reply:
No, those are percentile values. You're thinking damage_coeff which does in fact make them invulnerable when set to 0